src/test/java/org/broadinstitute/consent/http/service/dao/DraftServiceDAOTest.java

```diff
@@ -0,0 +1,242 @@
+    User user = createUser();
+    User user2 = createUser();
+    createDraft(user, 3);
+    createDraft(user2, 1);
+    createDraft(user2, 4);
+    assertThat(draftServiceDAO.findDraftsForUser(user2), hasSize(2));
+    assertThat(draftServiceDAO.findDraftsForUser(user), hasSize(1));
+    draftServiceDAO.deleteDraftsByUser(user2);
+    assertThat(draftServiceDAO.findDraftsForUser(user), hasSize(1));
+    assertThat(draftServiceDAO.findDraftsForUser(user2), hasSize(0));
+    draftServiceDAO.deleteDraftsByUser(user2);
+  }
+
+  @Test
+  void testAddAttachmentToDraft() throws SQLException, IOException {
+    when(gcsService.storeDocument(any(), anyString(), any())).thenReturn(
+        BlobId.of(UUID.randomUUID().toString(), UUID.randomUUID().toString()));
+    User user = createUser();
+    DraftInterface draft = createDraft(user, 3);
+    Map<String, FormDataBodyPart> files = getRandomFiles(1);
+    List<FileStorageObject> addedAttachments = draftServiceDAO.addAttachments(draft, user, files);
+    assertThat(addedAttachments, hasSize(1));
+    DraftInterface updatedDraft = draftServiceDAO.getAuthorizedDraft(draft.getUUID(), user);
+    assertThat(updatedDraft.getStoredFiles(), hasSize(4));
+  }
+
+  @Test
+  void testDeleteAttachmentFromDraft() throws SQLException, IOException {
+    when(gcsService.storeDocument(any(), anyString(), any())).thenReturn(
+        BlobId.of(UUID.randomUUID().toString(), UUID.randomUUID().toString()));
+    User user = createUser();
+    DraftInterface draft = createDraft(user, 3);
+    Set<FileStorageObject> storedFiles = draft.getStoredFiles();
+    assertThat(storedFiles, hasSize(3));
+    for (FileStorageObject file : storedFiles) {
+      draftServiceDAO.deleteDraftAttachment(draft, user, file.getFileStorageObjectId());
+    }
+    assertThat(draftServiceDAO.getAuthorizedDraft(draft.getUUID(), user).getStoredFiles(),
+        hasSize(0));
+  }
+
+
+
+  @Test
+  void testUpdateDraft() throws SQLException {
+    User user = createUser();
+    DraftInterface draft = createDraft(user, 1);
+    String updatedJson = "{\"study\": \"My example study\"}";
+    String newDraftName = "My favorite draft";
+    String originalDocumentJson = draft.getJson();
+    Date originalDocumentDate = draft.getUpdateDate();
+    assertEquals(originalDocumentJson, draft.getJson());
+    assertEquals(originalDocumentDate, draft.getUpdateDate());
+    assertNotEquals(draft.getName(), newDraftName);
+    draft.setName(newDraftName);
+    draft.setJson(updatedJson);
+    DraftInterface updatedDraft = draftServiceDAO.updateDraft(draft, user);
+    assertEquals(draft.getUUID(), updatedDraft.getUUID());
+    assertEquals(newDraftName, updatedDraft.getName());
+    assertEquals(updatedJson, updatedDraft.getJson());
+    assertThinUser(updatedDraft.getCreateUser());
+    assertThinUser(updatedDraft.getUpdateUser());
+  }
+
+  @NotNull
+  private DraftInterface createDraft(User user, Integer numberOfFiles)
```

Comment:
Unless you need to store `null`, I would avoid using boxed types like `Integer`. Using `int` makes it clear to the reader that the value can never be `null`.
```suggestion
  private DraftInterface createDraft(User user, int numberOfFiles)
```